bin/plugin/commands/changelog.js

```diff
@@ -822,7 +819,7 @@ function getContributorMarkdownList( ftcPRs ) {
  * @return {IssuesListForRepoResponseItem[]} The sorted list of pull requests.
  */
 function sortByUsername( items ) {
-	return sortBy( items, ( item ) => item.user.login );
+	return sortBy( items, ( item ) => item.user.login.toLowerCase() );
```

Comment:
Sorting by lowercase required else those with capitals in usernames end up _before_ those which come alphabetically before them.